events/dvg_england.txt

```diff
@@ -0,0 +1,776 @@
+	on_opened_soundeffect = "event:/SFX/Events/europenorthamerica/gold_prospectors"
+
+	icon = "gfx/interface/icons/event_icons/event_map.dds"
+
+    duration = 3
+
+	option = {
+		name = dvg_england.13.a
+		
+		change_relations = {
+				country = c:ENG
+				value = -50
+		}
+		default_option = yes
+		
+		set_global_variable = anglo_summit_fail
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+	
+}
+
+dvg_england.14 = {  #England Unite
+	type = country_event
+    placement = ROOT
+
+    title = dvg_england.14.t
+    desc = dvg_england.14.d
+    flavor = dvg_england.14.f
+
+    event_image = {
+		video = "gfx/event_pictures/europenorthamerica_london_center.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/europenorthamerica/gold_prospectors"
+
+	icon = "gfx/interface/icons/event_icons/event_map.dds"
+
+    duration = 3
+
+	option = {
+		name = dvg_england.14.a
+		
+		add_loyalists = {
+			value = 0.15
+			
+		}
+		add_primary_culture = cu:british
+		
+		add_primary_culture = cu:beornian
+		
+		change_tag = ENG 
+		
+		ai_chance = {
+			base = 100
+		}
+	}
+}
+
+dvg_england.15 = { #Beornia wants to join England
+	type = country_event
+    placement = ROOT
+
+    title = dvg_england.15.t
+    desc = dvg_england.15.d
+    flavor = dvg_england.15.f
+
+    event_image = {
+		video = "gfx/event_pictures/europenorthamerica_gold_prospectors.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/europenorthamerica/gold_prospectors"
+
+	icon = "gfx/interface/icons/event_icons/event_map.dds"
+
+    duration = 3
+
+	option = {
+		name = dvg_england.15.a
+		
+		c:ENG = {
+			create_diplomatic_pact = {
+				country = c:BEO
+				type = protectorate
+			}
+		}
+		
+		
+		ai_chance = {
+			base = 40
+		}
+	}
+	option = {
+		name = dvg_england.15.b
+		
+		annex = c:BEO
+		
+		ai_chance = {
+			base = 40
+		}
+	}
+	
+	option = {
+		name = dvg_england.15.c
+		
+		c:ENG = {
+			create_diplomatic_pact = {
+				country = c:BEO
+				type = dominion
+			}
+		}
+		default_option = yes
+		
+		ai_chance = {
+			base = 20
+		}
+	}
+}
+
+dvg_england.16 = {  #Anglo-Customs
+	type = country_event
+    placement = ROOT
+
+    title = dvg_england.16.t
+    desc = dvg_england.16.d
+    flavor = dvg_england.16.f
+
+    event_image = {
+		video = "gfx/event_pictures/europenorthamerica_capitalists_meeting.bk2"
+	}
+
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	on_opened_soundeffect = "event:/SFX/Events/europenorthamerica/gold_prospectors"
+
+	icon = "gfx/interface/icons/event_icons/event_map.dds"
+
+    duration = 3
+
+	option = {
+		name = dvg_england.16.a
+		
+		c:BEO = {
+					
+				}
+		
+		ai_chance = {
+			base = 1
+		}
+	}
+	option = {
+		name = dvg_england.16.b
+		default_option = yes
+		
+		c:ENG = {
+			create_diplomatic_pact = {
+				country = c:BEO
+				type = customs_union
+			}
+		
+		
+			ai_chance = {

```

Comment:
Unknown effect ai_chance at  file: events/dvg_england.txt line: 435
It is at the wrong level here. Should be outside the c:ENG tag